src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcRouteHandler.cs

```diff
@@ -45,25 +44,18 @@ public Task RouteAsync(RouteContext context)
 
             EnsureServices(context.HttpContext);
```

Comment:
Rebase with dev? We recently removed this. It can also be removed from `MvcAttributeRouteHandler`.